config/ui-extensions/nats/general

```diff
@@ -0,0 +1,15 @@
+resource:
+  kind: NATS
+  group: operator.kyma-project.io
+  version: v1alpha1
+urlPath: nats
+category: Kyma
+name: NATS
+scope: namespace
+features:
+  actions:
+    disableCreate: false
+    disableDelete: false
```

Comment:
don't we want to have disable enabled? I see you disable create NATS too.
If you disable both, does it mean that you don't allow create nor delete, but only allow enable.